homeassistant/components/alexa/capabilities.py

```diff
@@ -752,10 +752,11 @@ def configuration(self):
                 supported_modes.append(thermostat_mode)
 
         preset_modes = self.entity.attributes.get(climate.ATTR_PRESET_MODES)
```

Comment:
this is a smaller change `preset_modes = self.entity.attributes.get(climate.ATTR_PRESET_MODES, [])` but OTOH maybe less readable?

Reply:
I've chosen not to, indeed, to make it more readable on what is going on here.